pkg/contracts/src/CVStrategy.sol

```diff
@@ -209,14 +220,15 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
         pointSystem = ip.pointSystem;
         pointConfig = ip.pointConfig;
         _minThresholdPoints = ip.minThresholdPoints;
+        arbitrableConfig = ip.arbitrableConfig;
+
```

Comment:
we should register the tribunal safe here

Reply:
It needs to be called apart if not we are going to breake the IArbitrator Interface by adding the RegisterSafe function

Reply:
Hmm can cast to SafeArbitrator and call the register, right now we can assume that we always have the safe arbitrator but later we can add an extra variable that defines if its Kleros or SafeArbitrator or other and register in consequences ? So we need another transaction when creating the pool ? Maybe not a good UX no ?

Reply:
But any case we still need RegisterSafe to be called somewhere :p
I would be in favor of having both way (on poolCreation and through setArbitrationConfig)

Reply:
Actually did you meant that RegisterSafe should be called from the UI directly and bypass strategy ? if yess wont work because we register the msg.sender as the associated arbitrable.